### ISSUE TITLE
Compression error when the compressed content size is greater than 2^window_sz2 and the contents are all 0.

### ISSUE THREAD

**Issue by banana-sun** [yawor/heatshrinkpy#3]
OS : WIN 10
python : 3.10.7
```
from heatshrinkpy.core import decode, encode

str=None
with open("src.txt", 'rb') as src:
    str = src.read()

bin = encode(str, window_sz2=12, lookahead_sz2=4)
debin = decode(bin, window_sz2=12, lookahead_sz2=4)

print(f"{bin.hex(' ')}, {debin}")
```
src.txt is a binary file of 8004 bytes with all zeros. According to the above code test, can not be compressed.

**Comment by banana-sun** [comment]
I think I have found the problem. I will submit a merge request. Please help review it, thank you.

**Comment by yawor** [comment]
Fixed with #4. I've also added a test for this exact situation.
Version 0.11.1 published to pypi.
